module/solutions/helper.js

```diff
@@ -1786,9 +1786,9 @@ module.exports = class SolutionsHelper {
             let checkIfUserProjectExistsQuery = {
               createdBy: userId,
               referenceFrom: constants.common.LINK,
-              link: link,
+              solutionId: privateProgramAndSolutionDetails.result,
```

Comment:
@VISHNUDAS-tunerlabs We should first check if a project is there for this user and then create pvt solution and program

Reply:
@aks30 we need the solutionId (private) to determine whether a private project already exists or not. Our approach involves first checking if a private solution has been created. If it exists, we can avoid creating any new solutions or programs. However, if a private solution does not exist, we proceed to create a new solution and program. By using the obtained solutionId, we can then verify if the user has indeed created a private project